pkg/build/util/util.go

```diff
@@ -307,3 +307,19 @@ func SafeForLoggingS2IConfig(config *s2iapi.Config) *s2iapi.Config {
 	newConfig.ScriptsURL, _ = s2iutil.SafeForLoggingURL(newConfig.ScriptsURL)
 	return &newConfig
 }
+
+// Gets the Env section of the build config regardless of the build strategy
+func GetEnvFromStrategy(build *buildapi.Build) *[]kapi.EnvVar {
```

Comment:
I refactored the updateBuildEnv from generator.go into pkg/build/util/util.go in the next push, but will open an issue for the pkg/build/admission/buildpodutil.go, pkg/build/admission/defaults/admission.go  ones